dolphin/coordination.py

```diff
@@ -30,7 +30,10 @@
 coordination_opts = [
     cfg.StrOpt('backend_url',
                default='redis://127.0.0.1:6379',
-               help='The back end URL to use for distributed coordination.')
+               help='The back end URL to use for distributed coordination.'),
+    cfg.IntOpt('expiration',
+               default=100,
```

Comment:
100 means 100 seconds? Could you plz add a comment to explain it?

Reply:
Add some info in comment